src/components/box/box.stories.mdx

```diff
@@ -374,27 +374,4 @@ If no `scrollVariant` prop is set then the scroll bar will resort to the browser
 
 ## Props
 
-### Style System
-
-<StyledSystemProps spacing color flexBox hasGapProps layout position noHeader />
-
-<ArgsTable
-  of={Box}
-  exclude={[
-    "as",
-    "ref",
-    "forwardedAs",
-    "theme"
-  ]}
-/>
-
-<SimpleArgsTable
-  rows={[
-    {
-      name: "tabIndex",
-      type: { summary: "number | string" },
-      description: "Allows a tabindex to be specified",
-      required: false,
-    },
-  ]}
-/>
+<StyledSystemProps of={Box} spacing color flexBox layout position noHeader />
```

Comment:
```suggestion
<StyledSystemProps of={Box} spacing flexBox layout position noHeader />
```
The colour-related props are duplicated in storybook, so given we are defining them ourselves rather than extending styled system's prop interface, this should address it?